source/Interfaces/MainInterface.hpp

```diff
@@ -0,0 +1,79 @@
+/**
+ * @author : Team - 3
+ * @date: 10/03/2023
+ * MainInterface class creates a window and displays the default maze grid
+ */
+
+#pragma once
+
+#include "SFML/Graphics.hpp"
+#include "SFML/Graphics/Transformable.hpp"
+#include <iostream>
+#include <vector>
+#include <sstream>
+#include "Button.hpp"
+
+#include "../core/Data.hpp"
+#include "../core/InterfaceBase.hpp"
+
+//#include "Menu.h"
+
+using namespace cse491;
```

Comment:
If you are going to have a `using namespace` in a header file, make sure to put it inside of another namespace (in this case in you`i_2D` namespace.  Otherwise it will put the contents of cse491 in the global namespace, not just for this file but also for every other file that includes this one.